.circleci/conditional_config.yml

```diff
@@ -365,16 +365,32 @@ jobs:
       - attach_workspace:
          at: .
       - run:
-          name: Merge and check coverage
+          name: Merge coverage
           command: |
             cd src/api
-            export COVERALLS_REPO_TOKEN=HWLJwfiFsKPGEOzfgllO3pP3rqV540Qt3
             bundle exec rake ci:simplecov_ci_merge
+      - run:
+          name: Upload coverage to codecov
+          command: |
+            cd src/api
+            curl -LOs https://uploader.codecov.io/latest/linux/codecov
+            chmod +x codecov
+            ./codecov -v -f coverage/coverage.xml
```

Comment:
@hennevogel there is not Token required for the codecov upload?

Reply:
nope